feat: add apt_sources_facts module

s3lph opened the pull request.

wickdChromosome commented on plugins/modules/apt_sources_facts.py:
```diff
@@ -0,0 +1,160 @@
+        from aptsources.sourceslist import SourcesList, Deb822SourceEntry, SourceEntry
+    except ImportError:
+        warnings = [
+            'adfinis.facts.apt_sources_facts was invoked on a system that is missing the python3-apt or python-apt' +
```
I think that as it is, this will result in a missing space:
"missing the python3-apt or python-aptpackage.  ansible_facts.apt_sources will be empty."

wickdChromosome commented on plugins/modules/apt_sources_facts.py:
```diff
@@ -0,0 +1,160 @@
+
+    # Load all .list and .sources files
+    sources = SourcesList()
+    for ent in os.listdir('/etc/apt/sources.list.d'):
```
Maybe the dir string could be a named variable

wickdChromosome commented on plugins/modules/apt_sources_facts.py:
```diff
@@ -0,0 +1,160 @@
+                uri=source.uri,
+                suites=source.suites,
+                components=source.comps,
+                architectures=source.architectures
```
Maybe a trailing comma here?
So architectures=source.architectures,

s3lph commented:
@wickdChromosome thanks for the review. I fixed the stuff you noted. However, during another round of testing against older Debian versions, I noticed that the plugin would fail in versions without Deb822 sources support. So I added a safeguard for it.

I have to say, the `if deb822 and isinstance(source, Deb822SourceEntry):` looks a bit shady, but actually works 🙄

wickdChromosome commented:
Cool ! LGTM
